packages/babel-preset-peregrine/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@magento/babel-preset-peregrine",
-  "version": "1.2.0",
+  "version": "1.3.0",
```

Comment:
@mikhaelbois Can all of these @magento/<package> (internal packages) be reverted back?  During release alpha package we update all needed package versions at once.

Reply:
We can, but some were forgotten last time and that's what caused issues with NPM 7+ because of the peer dependencies that are more strict.
I will revert them all and do a reminder when a release 13.

Reply:
Thanks @mikhaelbois . I will check this too. We cant continue with this as our release script does not have capability to identify any in-between release changes.

Reply:
@dpatil-magento I just pushed the revert.